src/features/iam/iamHelpers.ts

```diff
@@ -31,7 +32,38 @@ async function createDefaultProject(): Promise<void> {
   ext.explorerTree.refresh()
 }
 
+const renameProject = async ({ projectId }: { projectId: string }) => {
+  const project = await iamState.getProjectById(projectId)
+  if (!project) return
```

Comment:
if we don't have a command for this purpose this check has no sense because we can rename only by right click on existing project

Reply:
removed